app/controllers/concerns/idv/verify_info_concern.rb

```diff
@@ -21,8 +21,6 @@ def shared_update
       document_capture_session.requested_at = Time.zone.now
 
       idv_session.verify_info_step_document_capture_session_uuid = document_capture_session.uuid
-      idv_session.vendor_phone_confirmation = false
-      idv_session.user_phone_confirmation = false
```

Comment:
This is now handled by `clear_future_steps!` since these are set by the phone step.